### PR TITLE
Add Route optimisation

### PR DESCRIPTION
**Pull request by Sanketwable**
fixes https://github.com/bench-routes/bench-routes/issues/466

### CODE REVIEW

**Comment by Sanketwable** [comment]
@Harkishen-Singh @muskankhedia @aquibbaig Please review

File: src/lib/config/config.go [hunk 1]
```diff
@@ -122,6 +123,26 @@ func (inst *Config) Write() (bool, error) {
 
 	return true, nil
 }
+func (inst *Config) AppendWrite(route []Route) (bool, error) {
```
Comment (aquibbaig) [inline src/lib/config/config.go:126]: Interesting! Does this function return a boolean that is never used?

**Review by aquibbaig — APPROVED** [review]
Let's drop the boolean from the function and we're good to go!

**Review by Sanketwable — COMMENTED** [review]
Removed the boolean return

**Pull request closed without merging**